src/Hazelcast.Net.Tests/Serialization/Compact/CompactReaderWriterExtensionsTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#nullable enable
+
+using System;
+using Hazelcast.Core;
+using Hazelcast.Serialization;
+using Hazelcast.Serialization.Compact;
+using NUnit.Framework;
+
+namespace Hazelcast.Tests.Serialization.Compact;
+
+[TestFixture]
+public class CompactReaderWriterExtensionsTests
+{
+    [Test]
+    public void WriteAny()
+    {
+        var kinds = (FieldKind[])Enum.GetValues(typeof(FieldKind));
+        Assert.That(kinds[0], Is.EqualTo(FieldKind.NotAvailable));
+
+        var schemaBuilder = SchemaBuilder.For("test");
+        for (var i = 1; i < kinds.Length; i++)
+        {
+            schemaBuilder = schemaBuilder.WithField($"field-{kinds[i]}", kinds[i]);
+        }
+        var schema = schemaBuilder.Build();
+
+        var orw = new ReflectionSerializerTests.ObjectReaderWriter(new ReflectionSerializer());
+        var output = new ObjectDataOutput(1024, orw, Endianness.LittleEndian);
```

Comment:
Just asking. Are `{ }` valid characters for a field name?

Reply:
I haven't been able to find a spec for valid field names but I would say it's `a-z`, `A-Z`, `0-9` and `_`. I don't expect anything else to be supported.

Reply:
Sorry, it's my mistake. In my editor, I couldn't see that curly brackets are the part of string interpolation 😓